client/flags/flags.go

```diff
@@ -15,7 +15,7 @@ const (
 	// failures due to state changes that might occur between the tx simulation
 	// and the actual run.
 	DefaultGasAdjustment = 1.0
-	DefaultGasLimit      = 200000
+	DefaultGasLimit      = 250000
```

Comment:
@likhita-809 Do you know how many tests are failing if we roll back to 200000?
- if it's only 2-3 tests, let's fix them manually by passing `--gas-limit=250000`
- if it's >10 tess, probably it's a good idea to bump to 250000, but let's also add a CLI-breaking change changelog entry

Reply:
I think only 1 test is failing when gas-limit is 200000, maybe I'll try fixing it by passing `--gas-limit` flag.